app/controllers/quizzes_controller.rb

```diff
@@ -2,46 +2,46 @@ class QuizzesController < ApplicationController
     def index
       @quizzes = Quiz.all
     end
-  
+
     def show
       @quiz = Quiz.find(params[:id])
     end
-  
+
     def new
       @quiz = Quiz.new
     end
-  
+
     def create
       @quiz = Quiz.new(quiz_params)
       if @quiz.save
-        redirect_to @quiz
+        redirect_to @quiz, notice: "Quiz was successfully created."
       else
         render :new
       end
     end
-  
+
     def edit
       @quiz = Quiz.find(params[:id])
     end
-  
+
     def update
       @quiz = Quiz.find(params[:id])
       if @quiz.update(quiz_params)
-        redirect_to @quiz
+        redirect_to @quiz, notice: "Quiz was successfully updated."
       else
         render :edit
       end
     end
-  
+
     def destroy
       @quiz = Quiz.find(params[:id])
       @quiz.destroy
-      redirect_to quizzes_path
-    end
-  
+    redirect_to quizzes_url, notice: "Quiz was successfully destroyed."
```

Comment:
Your indentation here looks a little off, I think the file as a whole is a bit off as a whole. I think it's because the linter that your app has come configured with has very few rules. 

The best quick fix for having the linter keep on top of this is to add: 

```
Layout/IndentationConsistency:
  Enabled: true

Layout/IndentationWidth:
  Enabled: true
```

to the bottom of your .rubocop.yml file.

A more longer term fix would be to use the rubocop-govuk rules. If that would be helpful I'd be happy to open a PR that adds that to this project?